tests/foreman/ui/test_contentview.py

```diff
@@ -31,14 +31,18 @@
     upload_manifest,
     get_role_by_bz,
 )
+from robottelo.cli.activationkey import ActivationKey
 from robottelo.cli.contentview import ContentView
 from robottelo.cli.factory import (
+    activationkey_add_subscription_to_repo,
     make_content_view,
     make_content_view_filter,
     make_content_view_filter_rule,
+    make_lifecycle_environment as cli_make_lifecycle_environment,
```

Comment:
any reason for that change? Looks really weird

Reply:
yep, we've got ui factory helper with exactly the same name

Reply:
i was actually thinking about adding `cli_` prefix to all cli factory helpers here, but i'm not sure as it will make all the helpers look weird as you've pointed :) right now only lce helper is conflicting so only this change is a must.